bcda/auth/plugin/alpha.go

```diff
@@ -0,0 +1,43 @@
+package auth
+
+import (
+	"errors"
+)
+
+type AlphaAuthPlugin struct{}
+
+func (p *AlphaAuthPlugin) RegisterClient(params []byte) error {
+	return errors.New("Not yet implemented")
+}
+
+func (p *AlphaAuthPlugin) UpdateClient(params []byte) error {
+	return errors.New("Not yet implemented")
+}
+
+func (p *AlphaAuthPlugin) DeleteClient(params []byte) error {
+	return errors.New("Not yet implemented")
+}
+
+func (p *AlphaAuthPlugin) GenerateClientCredentials(params []byte) ([]byte, error) {
+	return nil, errors.New("Not yet implemented")
+}
+
+func (p *AlphaAuthPlugin) RevokeClientCredentials(params []byte) error {
+	return errors.New("Not yet implemented")
+}
+
+func (p *AlphaAuthPlugin) RequestAccessToken(params []byte) ([]byte, error) {
+	return nil, errors.New("Not yet implemented")
+}
+
+func (p *AlphaAuthPlugin) RevokeAccessToken(params []byte) error {
```

Comment:
RevokeAccessToken(token string) error 

Tokens are passed around the net as base64 encoded strings. Internally, these methods will decode the string, validate the token, and then do the thing. Where the type is as well known as this, I think it's worth being explicit.